test/smtcallback.js

```diff
@@ -34,7 +34,12 @@ function expectErrors (errors, expectations) {
     if (errors[i].includes('Error trying to invoke SMT solver') || expectations[i].includes('Error trying to invoke SMT solver')) {
       continue;
     }
-    expectations[i] = expectations[i].replace('happens here.', 'happens here');
+    expectations[i] = expectations[i].replace('happens here.', 'happens here').split('\\n').join('\n');
+    errors[i] = errors[i].replace('happens here.', 'happens here').split('\\n').join('\n');
+    if (ignoreCex) {
+      expectations[i] = expectations[i].split('\n')[0];
```

Comment:
I don't fully understand why is it done this way. Why not just cut off anything after `Counterexamples:` ?

Reply:
Updated